src/os/TempFile.cxx

```diff
@@ -43,7 +43,8 @@
 
 #include "os/TempFile.hxx"
 
-#ifndef __FreeRTOS__
+/// @todo mingw does not seem to have an mkdtemp call.
+#if !defined(__FreeRTOS__) && !defined(__WINNT__)
```

Comment:
Could we handle this through openmrn_features.h?

Reply:
Where is this used? We may want to exclude it entirely for some envs beyond the two above

Reply:
i moved this change out of this PR because it is unrelated.